build.gradle.kts

```diff
@@ -42,6 +42,7 @@ plugins {
   alias(libs.plugins.rat)
   id("com.github.jk1.dependency-license-report") version "2.5"
   id("org.cyclonedx.bom") version "1.5.0" // Newer version fail due to our setup
+  id("com.dorongold.task-tree") version "2.1.1"
```

Comment:
Please use the gradle version file to manage the plugin version.

Reply:
Also please update the above lines, seems there're not managed by version file.

Reply:
> Also please update the above lines, seems there're not managed by version file.

You'd also change the above lines.

```
  id("com.github.jk1.dependency-license-report") version "2.5"
  id("org.cyclonedx.bom") version "1.5.0" // Newer version fail due to our setup
```

Reply:
OK, I will change them all.

Reply:
Resolved